File: targets/java-spring/config/AppConfig.java
package com.baddemo.config;

/** Application configuration. Secrets are hardcoded on purpose (demo target). */
public class AppConfig {
    public static final String JWT_SECRET = "secret";
    public static final String DB_URL = "jdbc:mysql://localhost:3306/baddemo";
    public static final String DB_USER = "admin";
    public static final String DB_PASSWORD = "admin123";
    public static final String UPLOAD_DIR = "/var/www/uploads";
}

File: targets/csharp-aspnet/Config/AppConfig.cs
namespace BadDemo.Config
{
    /// <summary>Application configuration and shared constants.</summary>
    public static class AppConfig
    {
        public const string JwtSecret = "secret";
        public const string ConnectionString = "Server=localhost;Database=baddemo;User=admin;Password=admin123;";
        public const string UploadDir = "/var/www/uploads";
    }
}
